bulk-ingester/src/integration-test/java/org/alfresco/hxi_connector/bulk_ingester/processor/DummyIngestEventPublisher.java

```diff
@@ -0,0 +1,76 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.bulk_ingester.processor;
+
+import static java.lang.String.format;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.alfresco.hxi_connector.bulk_ingester.event.IngestEventPublisher;
+import org.alfresco.hxi_connector.common.model.ingest.IngestEvent;
+
+@Slf4j
+public class DummyIngestEventPublisher implements IngestEventPublisher
+{
+    private final Map<String, IngestEvent> ingestEvents = new HashMap<>();
```

Comment:
I think this is only used by one test, but it might be good to reset it between tests to try to avoid dirty data.

Reply:
After my changes this one is used by 2 classes - and thanks - I'll do a clean up (even between each method).